datacube/utils/geometry/_base.py

```diff
@@ -853,9 +853,6 @@ def multipoint(coords: CoordList, crs: MaybeCRS) -> Geometry:
     """
     Create a 2D MultiPoint Geometry
 
-    >>> multipoint([(10, 10), (20, 20)], None)
-    Geometry(MULTIPOINT (10 10, 20 20), None)
-
```

Comment:
This test fails - looks like shapely has changed the str representation of multipoint geometries.